README.md

```diff
@@ -46,18 +48,13 @@ Provides functions to create [RxJS](https://github.com/ReactiveX/RxJS) subjects
 
 ## Usage
 
-`enchannel-zmq-backend` exports four functions to create channels
+`enchannel-zmq-backend` exports a `createChannels` method for creating a channels object per [enchannel spec](https://github.com/nteract/enchannel).
```

Comment:
I'd love to re-phrase this as:

> To get access to all of the `channels` for messaging, use `createChannels`:

After Travis passes and I merge I'll follow up.